LiskHQ/lisk-sdk
# Fix P2P endpoint path - Closes #4646

### What was the problem?
P2P was using the default path(socketcluster) for the connection

### How did I solve it?
Pass the path parameter.

### How to manually test it?
- Observe that the network can be created.
- Observe that connection path is IP:port/rpc

### Review checklist

- [ ] The PR resolves #4646
